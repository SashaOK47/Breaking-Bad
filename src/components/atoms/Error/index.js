import PropTypes from "prop-types";
import style from "./styles.module.scss";

const Error = ({ message }) => {
  return <div className={style.error}>{message}</div>;
};
Error.propTypes = {
  message: PropTypes.string,
};

export default Error;
