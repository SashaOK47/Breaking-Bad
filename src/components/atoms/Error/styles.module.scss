@import "../../../assets/styles/main.scss";
.error {
  text-align: center;
  font-size: 3rem;
  color: $colorError;
}
