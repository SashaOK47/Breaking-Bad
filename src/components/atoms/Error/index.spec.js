import React from "react";
import Error from "./index";

const setUp = (props) => shallow(<Error {...props} />);
const message = "Test Error";

describe("Error component", () => {
  it("should render Error component", () => {
    const component = setUp();
    expect(component).toMatchSnapshot();
  });
  it("should render Error component message", () => {
    const component = setUp({ message });
    expect(component).toMatchSnapshot();
  });
});
