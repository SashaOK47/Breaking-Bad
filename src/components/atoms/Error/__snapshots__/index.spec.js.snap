// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render Error component 1`] = `
<div
  className="error"
/>
`;

exports[`Error component should render Error component message 1`] = `
<div
  className="error"
>
  Test Error
</div>
`;
